src/domains/ecommerce-app/api/ecommerce-webview/v2/_openapi.json.tpl

```diff
@@ -0,0 +1,456 @@
+          },
+          "status": {
+            "$ref": "#/components/schemas/HttpStatusCode"
+          },
+          "detail": {
+            "type": "string",
+            "description": "A human readable explanation specific to this occurrence of the\nproblem.",
+            "example": "There was an error processing the request"
+          },
+          "instance": {
+            "type": "string",
+            "format": "uri",
+            "description": "An absolute URI that identifies the specific occurrence of the problem.\nIt may or may not yield further information if dereferenced."
+          }
+        }
+      },
+      "HttpStatusCode": {
+        "type": "integer",
+        "format": "int32",
+        "description": "The HTTP status code generated by the origin server for this occurrence\nof the problem.",
+        "minimum": 100,
+        "maximum": 600,
+        "exclusiveMaximum": true,
+        "example": 500
+      },
+      "RptId": {
+        "type": "string",
+        "pattern": "([a-zA-Z0-9]{1,35})|(RFd{2}[a-zA-Z0-9]{1,21})"
+      },
+      "PaymentInfo": {
+        "description": "Informations about transaction payments",
+        "type": "object",
+        "properties": {
+          "paymentToken": {
+            "type": "string"
+          },
+          "rptId": {
+            "$ref": "#/components/schemas/RptId"
+          },
+          "reason": {
+            "type": "string"
+          },
+          "amount": {
+            "$ref": "#/components/schemas/AmountEuroCents"
+          },
+          "isAllCCP": {
+            "description": "Flag for the inclusion of Poste bundles. false -> excluded, true -> included",
+            "type": "boolean"
+          },
+          "transferList": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Transfer"
+            },
+            "minItems": 1,
+            "maxItems": 5
+          }
+        },
+        "required": [
+          "rptId",
+          "amount"
+        ],
+        "example": {
+          "rptId": "77777777777302012387654312384",
+          "paymentToken": "paymentToken1",
+          "reason": "reason1",
+          "amount": 600,
+          "authToken": "authToken1",
+          "isAllCCP": false,
+          "transferList": [
+            {
+              "paFiscalCode": "77777777777",
+              "digitalStamp": false,
+              "transferCategory": "transferCategory1",
+              "transferAmount": 500
+            },
+            {
+              "paFiscalCode": "11111111111",
+              "digitalStamp": true,
+              "transferCategory": "transferCategory2",
+              "transferAmount": 100
+            }
+          ]
+        }
+      },
+      "NewTransactionResponse": {
```

Comment:
remove this unused object

Reply:
done here 254f7eeac6e95e525750dfb6acff440202a34091